boucle.py

```diff
@@ -163,24 +183,27 @@ def my_callback(frames):
 
     return jack.CALL_AGAIN
 
+
 client.set_process_callback(my_callback)
 
 # activate !
 with client:
-
     # make connection
     playback = client.get_ports(is_physical=True, is_input=True)
     if not playback:
         raise RuntimeError("No physical playback ports")
 
-    client.connect(outL, playback[0])
-    client.connect(outR, playback[1])
-
     record = client.get_ports(is_physical=True, is_output=True)
     if not record:
         raise RuntimeError("No physical record ports")
 
-    client.connect(record[0], inL)
-    client.connect(record[1], inR)
+    # if user has other connections that standard Main Out (i.e complex connections with dedicated outputs),
+    # then it is much better to leave the user creates his connections with jack patchbay instead of connecting automatically here.
+    if gui.auto_connect:
```

Comment:
If people need some other outputs than default MAIN, I think they will known about jack routing.  (maybe will can camelcase default name : Main)
We can add section in readme about ports, routing and connections :

    ### Audio output ports

    By default, superboucle have one stereo ouput called 'Main'. You can
    add other output port with 'add port...' entry in dropbox. At startup, 
    Superboucle will only connect 'Main' output port to the first stereo 
    pair found on soundcard.